electron/renderer/src/containers/TeamsSidebar.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux'
+
+import { addAccountWithSession, switchAccount } from '../actions'
+import Sidebar from '../components/Sidebar'
+
+const mapStateToProps = (state) => {
+  return {
+    accounts: state.accounts
+  }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    onAddAccountClick: () => {
+      dispatch(addAccountWithSession())
+    },
+    onSwitchAccountClick: (id) => {
+      dispatch(switchAccount(id))
+    }
+  }
+}
+
+const TeamsSidebar = connect(
```

Comment:
WDYT about moving the connect to the Sidebar component since it has already team specific elements like team icon and variable names like `account`? Do you prefer to have it separated to have different types of sidebars?